### PR TITLE
configurable robots.txt in dataverse payara deployment

### PR DESCRIPTION
**Pull request by pallinger**
you can define dataverse.robots_txt_url which overwrites robots.txt in the dataverse payara deployment

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -257,6 +257,7 @@ dataverse:
     argument: '-DcompilerArgument=-Xlint:unchecked test -P all-unit-tests'
   usermgmtkey: burrito
   version: '5.13'
+#  robots_txt_url: https://guides.dataverse.org/en/latest/_downloads/3a5cd7a283eecd5e93289e30af713554/robots.txt
```
Comment (pdurbin) [inline defaults/main.yml:260]: I'm not sure if this is our usual practice... to comment out variables like this. But I guess it's fine.

**Review by pdurbin — COMMENTED** [review]
Seems like nice functionality. I left a comment about style.

File: defaults/main.yml (hunk 1 above)
Comment (pallinger) [reply defaults/main.yml:260]: Where would you put the address of the recommended robots.txt ?
There are some similar lines in defaults/main.yml, also for optional features. Some of them added by me :)

Comment (pdurbin) [reply defaults/main.yml:260]: Hmm, I'm not sure. What you're doing is nice and compact. 😅

Comment (pallinger) [reply defaults/main.yml:260]: okay, then I will merge this...

**Pull request merged**